go.mod

```diff
@@ -3,23 +3,27 @@ module github.com/rancher/vm
 go 1.13
 
 replace (
+	github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.2.0+incompatible
 	github.com/crewjam/saml => github.com/rancher/saml v0.0.0-20180713225824-ce1532152fde
-	github.com/rancher/apiserver => github.com/rancheredge/apiserver v0.0.0-20200708064345-bd2aa8f9b7d1
```

Comment:
Why using native api server instead fixed apiserver in `rancheredge` repo?

Reply:
To have upstream fixes. e.g. https://github.com/rancher/apiserver/commit/52a54b92238638ae82615a563c0fb1bdc21bc51b